VAMobile/src/screens/HealthScreen/Appointments/AppointmentDetailsCommon/PreferredAppointmentType.tsx

```diff
@@ -30,7 +30,7 @@ function PreferredAppointmentType({ attributes }: PreferredAppointmentTypeProps)
       case AppointmentTypeConstants.VA_VIDEO_CONNECT_HOME:
       case AppointmentTypeConstants.VA_VIDEO_CONNECT_ONSITE:
       case AppointmentTypeConstants.VA_VIDEO_CONNECT_GFE:
-        preferredTypeofAppointment = t('appointments.pending.video')
+        preferredTypeofAppointment = t('video')
```

Comment:
if mine gets merged first then there will be a conflict with this string, I didn't change any names in my branch I believe

Reply:
just calling it out so we don't forget once either is merged

Reply:
Makes sense, shouldn't be a hard conflict to fix